src/__tests__/wait-for.js

```diff
@@ -292,12 +292,11 @@ test('the real timers => fake timers error shows the original stack trace when c
 
 test('does not work after it resolves', async () => {
```

Comment:
I made some changes on this test to help understand the outcome: everything that starts needs to end